edx-platform/pearson-theme/lms/static/sass/partials/lms/theme/_dashboard.scss

```diff
@@ -81,29 +81,26 @@ div.my-courses div.empty-dashboard-message {
     width: 100%;
   }
 
-  .course .details .wrapper-course-details {
+  .course .details .wrapper-course-details  {
```

Comment:
I see in the code sometimes you use 0,1 or two spaces between the selector and the curly braces, can we use a standard, leaving only one space? This is to have an uniform code